wiki/People/Nomination_Assessment_Team/en.md

```diff
@@ -34,7 +34,7 @@ NAT members responsible for this category are in high demand due to the sheer vo
 NAT members assigned to the structural category are not responsible for any specific game mode, despite being allowed to nominate beatmaps of their previously assigned game mode. They are required to maintain:
 
 - **Communication**: Promoting transparency and maintaining good relations with not only the rest of the mapping/modding community, but also within the NAT itself. This includes, but not limited to, making announcements, participating in discussions about proposals, asking/answering questions through surveys, and updating the [ranking criteria](/wiki/Ranking_criteria) as well as other documentation.
-- **Development**: Developing and maintaining tools and websites to help improve the ranking process (such as [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier), [Aiess](https://github.com/Naxesss/Aiess), or [the NAT/BN website](https://bn.mappersguild.com/home)).
+- **Development**: Developing and maintaining tools and websites to help improve the ranking process (such as [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier), [Aiess](https://github.com/Naxesss/Aiess), or [the BN Management website](https://bn.mappersguild.com)).
```

Comment:
```suggestion
- **Development**: Developing and maintaining tools and websites to help improve the ranking process (such as [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier), [Aiess](https://github.com/Naxesss/Aiess), or [the BN management website](https://bn.mappersguild.com)).
```

Reply:
https://github.com/ppy/osu-wiki/pull/10092#discussion_r1335047275

Reply:
time is a flat circle